stovermc/grab_bag
# Drop box oauth

@jmejia  This PR completes my first user story and implements basic function on my second user story. The second user story is not yet tested...
 
#### Fixes #
Completes #25 
Completes #15 
Begins implementation #16 

#### What does  this PR do?

#### Where should the reviewer start?
bundle
rake db:reset
check rspec (relevant tests are passing, a few are skipped due to other stories not addressed here)

#### How should this be manually tested?
Rails s
Click on "connect to dropbox"
You should see your dropbox account info
You should be able to click on a FILE (only files right now work) and it will copy over to your grabbag account 

#### Any background context you want to provide?

#### Screenshots (if appropriate)

#### Questions:
  - Do Migrations Need to be ran? 
  reset db
  - Do Environment Variables need to be set? 
  make sure dropbox keys are set
  - Any other deploy steps?
